packages/lib/src/language/locales/en-US.json

```diff
@@ -135,6 +135,8 @@
     "select.stateOrProvince": "Select state or province",
     "select.provinceOrTerritory": "Select province or territory",
     "select.country": "Select country",
+    "select.noOptionsFound": "No options found",
+    "select.filter.placeholder": "Search...",
```

Comment:
Reminder: these need to be added into the translations repository

Reply:
I've created the translation job.